### DISCUSSION TITLE
"make auth" does not create TEST_APP_ID , TEST_USER_NAME , TEST_USER_PASSWORD , TEST_WORKSPACE_APP_ID , TEST_WORKSPACE_APP_SECRET for Github Actions

### DISCUSSION THREAD

**Discussion by jzcwong** [microsoft/AzureTRE#4019] [Q&A]
Hi team,
I'm trying to use GH actions to deploy AzureTRE, from https://microsoft.github.io/AzureTRE/v0.16.0/tre-admins/setup-instructions/workflows/ it seems like there are additional environment secrets/variables that are needed but running "make auth" does not populate these values in config.yaml.

The instructions to create TEST_APP_ID , TEST_USER_NAME , TEST_USER_PASSWORD , TEST_WORKSPACE_APP_ID , TEST_WORKSPACE_APP_SECRET are not clear in terms of what permissions need to be assigned to these service principals and user? 

Appreciate some guidance on this.

**Comment by West-P** [comment]
Also having the same issue here. No documentation on TEST_APP_ID, yet after following all steps to setup a CI/CD deployment in Github the Deploy TRE workflow gives me the following error:

![image](https://github.com/user-attachments/assets/a34f4ab8-ec44-4d9c-aaac-28d54ee4abe7)
